src/pages/QueryBuilder.vue

```diff
@@ -205,7 +205,7 @@
 </template>
 
 <script lang="ts">
-import { DataSource, DataType, EntityType, ExpressionFunction, Phenotype, Query, QueryPage, TypeEnum } from '@onto-med/top-api'
+import { DataSource, DataType, EntityType, ExpressionFunction, Phenotype, PhenotypeQuery, Query, QueryPage, TypeEnum } from '@onto-med/top-api'
```

Comment:
```suggestion
import { DataSource, DataType, EntityType, ExpressionFunction, Phenotype, PhenotypeQuery, QueryPage, TypeEnum } from '@onto-med/top-api'
```

I Think 'Query' import can be removed here. There may be other occurrences in this file that need to be updated accordingly.